views/layout.hbs

```diff
@@ -14,7 +14,8 @@
     <div class="container">
       {{{body}}}
     </div>
-
+<a href="/movies">List of movies</a>
+<a href="/celebrities">List of celebrities</a>
```

Comment:
Possibly the navbar should be on the top of the page and also it would be good to have links to navigate to the create new movie / celebrety